src/sqlite-xamarin/sqlite-xamarin.mdproj

```diff
@@ -0,0 +1,60 @@
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ItemType>GenericProject</ItemType>
+    <ProjectGuid>{B8F799C5-D7CE-4E09-9CE6-BAA4173E7EC8}</ProjectGuid>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)'=='Debug'">
+    <OutputPath>..\..\bin\Debug\</OutputPath>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)'=='Release'">
+    <OutputPath>..\..\bin\Release\</OutputPath>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <BuildDependsOn>
+        _CreateProject;
+        _NdkBuild;
+    </BuildDependsOn>
+  </PropertyGroup>
+
+  <Import Project="..\..\Configuration.props" />
+
+  <!-- Build target and co. -->
+
+  <Target Name="Build" DependsOnTargets="$(BuildDependsOn)" />
+
+  <Target Name="_CreateProject">
+    <Exec Condition="!Exists('src')"
+        Command="$(ANDROID_SDK_PATH)\tools\android create project -g -n sqlitesample -p . -a MainActivity -t android-10 --package com.xamarin.sqlitesample -v 1.0.0" />
```

Comment:
This should use `$(AndroidSdkDirectory)`, not `$(ANDROID_SDK_PATH)`. `$(AndroidSdkDirectory)` is set in `Configuration.props`, and thus is always present. `$(ANDROID_SDK_PATH)` might not be set.